substrate/primitives/api/src/lib.rs

```diff
@@ -127,6 +127,7 @@ use sp_runtime::{traits::Block as BlockT, ExtrinsicInclusionMode};
 pub use sp_state_machine::StorageProof;
 #[cfg(feature = "std")]
 use sp_state_machine::{backend::AsTrieBackend, Backend as StateBackend, OverlayedChanges};
+use sp_std::vec::Vec;
```

Comment:
This leads to a compile error for me; remove and use `alloc::vec::Vec` in the change below?

Reply:
I've removed it...there are other `Vec`s here which are not fully qualified and they are fine.

Reply:
Yeah, the other non fully qualified ones are all behind `std` iirc which is why they were ok :)

Reply:
Ah yeah :facepalm: 

Added it back using `sp_std` since that is handling aliasing to `alloc::vec` for consistency with how it is done elsewhere.